x-pack/plugins/apm/common/environment_filter_values.ts

```diff
@@ -43,18 +44,30 @@ export const ENVIRONMENT_NOT_DEFINED = {
   label: getEnvironmentLabel(ENVIRONMENT_NOT_DEFINED_VALUE),
 };
 
-export function getEnvironmentEsField(environment: string) {
-  if (
+function isEnvironmentDefined(environment: string) {
```

Comment:
@kpatticha Shouldn't this be called `isEnvironmentUndefined`?

Reply:
ah! I fixed it and forgot to push 😢 
<img width="627" alt="image" src="https://user-images.githubusercontent.com/3369346/215184384-263a1599-155e-4067-8e68-b9695ecba9fd.png">

thanks for catching this!, will fix it in a follow-up PR